manage_breast_screening/mammograms/views/appointment_note_views.py

```diff
@@ -78,3 +78,15 @@ def get_object(self):
 
     def get_success_url(self):
         return reverse("mammograms:appointment_note", kwargs={"pk": self.kwargs["pk"]})
+
```

Comment:
Can this logic be moved into the superclass do you think? I've just added something similar for add and update in this PR https://github.com/NHSDigital/dtos-manage-breast-screening/pull/824